src/documents/ComponentConfigurationOptions.md

```diff
@@ -4,30 +4,28 @@ layout: default
 category: Infusion
 ---
 
-Infusion components are configured using options that are defined by the component developer and customized by the integrator. While component developers are free to define whatever options are appropriate for their component, the Infusion Framework supports a number of predefined options. This page briefly describes these predefined options and provides links more information about the related Framework functionality.
+Infusion components are configured using options that are defined by the component developer and customized by the integrator. 
+While component developers are free to define whatever options are appropriate for their component, the Infusion Framework supports a number of predefined options.
 
-Some predefined options should not be overridden by integrators: They are strictly for the use of the component developer. This is noted in the descriptions below.
+The particular set of options interpreted by the framework is determined by the [Grades](ComponentGrades.md) that the component is derived from. Developers and integrators
```

Comment:
The first sentence of this paragraph implies that the framework will only interpret options defined by the specified grades, and not by the current component.